rw/internal/platform/rates/exchangeapi/client_test.go

```diff
@@ -0,0 +1,46 @@
+package exchangeapi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewClient(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		url string
+	}
+	tests := []struct {
+		name string
+		args args
+		want Client
+	}{
+		{
+			name: "Should initiate new client correctly",
+			args: args{
+				url: "https://url.com",
+			},
+			want: Client{
+				url: "https://url.com",
+			},
+		},
+		{
+			name: "Should initiate new client correctly",
+			args: args{
+				url: "https://url2.com",
+			},
+			want: Client{
+				url: "https://url2.com",
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			got := NewClient(tt.args.url)
+			require.Equal(t, tt.want, got)
+		})
+	}
+}
```

Comment:
OPTIONAL: is whole table test setup required for such a simple test?

Reply:
@Hukyl yes. In the future, it will simplify adding new test cases.
For now, this `NewClient` function is a simple constructor, that does nothing, but return a pointer to a struct. But, what if it will contain more complicated logic in the future? Of course, it will require adding more test cases.